action.yml

```diff
@@ -4,37 +4,52 @@ branding:
   icon: "image"
   color: "orange"
 inputs:
-  github-token:
-    description: "GitHub token"
-    required: true
   pizza-args:
     description: "OpenSauced Pizza CLI command to run"
     default: "generate codeowners ./"
     required: true
+  commit-and-push:
+    description: "Commit and push changes"
+    default: "false"
+    required: false
+
 runs:
   using: "composite"
   steps:
     - name: Checkout
       uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
 
-    - name: Setup Node
-      uses: actions/setup-node@v3
+    - name: Setup Go
+      uses: actions/setup-go@v4
       with:
-        node-version: "20"
+        go-version: "1.22.5"
+
+    - name: Install pizza CLI
+      run: |
+        go install github.com/open-sauced/pizza-cli@beta
+      shell: bash
 
-    - name: Install dependencies
-      run: npm ci
+    - name: Check pizza CLI version
+      run: pizza-cli version
```

Comment:
I left this as 	pizza-cli` for now @jpmcb because `go install` doesn't support it and I wanted to install a version instead of build it where I could use `-o`.